firestore/incidents.py

```diff
@@ -18,37 +18,45 @@ class Incident(mdl.Model):
     created_by = mdl.TextField(required=False)
     title = mdl.TextField(required=True)
     title_translate = mdl.MapField(required=False)
-    publish_status = mdl.MapField(required=True, default={
-        'twitter' : None,
-        'linkedin' : None
-    })
-    donation_link = mdl.TextField() #  Donation: link to donation website
-    police_tip_line = mdl.TextField() # Police Tip Line: phone number to provide tips to police to help capture the suspect
-    help_the_victim = mdl.TextField() # Help the victim: other free style text about how people can help the victim
+    publish_status = mdl.MapField(
+        required=True, default={"twitter": None, "linkedin": None}
+    )
+    donation_link = mdl.TextField()  #  Donation: link to donation website
+    police_tip_line = (
+        mdl.TextField()
+    )  # Police Tip Line: phone number to provide tips to police to help capture the suspect
+    help_the_victim = (
+        mdl.TextField()
+    )  # Help the victim: other free style text about how people can help the victim
+    parent_doc = mdl.TextField(column_name="parent")
```

Comment:
Are you sure this will work?

Reply:
Ref: https://github.com/octabytes/FireO/issues/20#issuecomment-556989359